moja-kuhinjica-frontend/src/components/button/AmountButton.tsx

```diff
@@ -1,22 +1,50 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Image from 'next/image'
 import increment from 'public/static/assets/images/increment.svg'
 import decrement from 'public/static/assets/images/decrement.svg'
 import styles from './AmountButton.module.scss'
+import { useAppDispatch } from '@/utils/hooks'
+import { changeMealAmount } from '@/reduxStore/reducers/restaurantReducer'
+import { IMeal } from '@/service/Restaurant.service'
 
 interface IAmountButtonProps {
     style?: string
     labelStyle?: string
+    meal: IMeal
 }
 export const AmountButton = ({
     style,
     labelStyle,
+    meal,
 }: IAmountButtonProps): JSX.Element => {
+    const [amount, setAmount] = useState<number>(1)
```

Comment:
add 1 to const for example on top file `const FIRST_ELEMENT = 1;`

Reply:
Done.